jenetics.doc/src/main/java/Knapsack.java

```diff
@@ -57,7 +57,7 @@ static Item random() {
 	fitness(final double size) {
 		return items -> {
 			final Item sum = items.stream().collect(Item.toSum());
-			return sum.size <= size ? sum.value : 0;
```

Comment:
I  think this works as expected. I check if the knapsack is not brimmed over (`sum.size <= size`). In this case the value of the knapsack (`sum.value`) is returned. A brimmed over knapsack has a fitness value of zero.

Reply:
Thanks for your answer.
However, currently the result is an overfilled knapsack.

![image](https://user-images.githubusercontent.com/6312834/107074079-849eb800-67e8-11eb-9953-285040999da6.png)

Reply:
The aim is to maximize the _value_ of the knapsack, without overfilling it. I think what you want to print is the following:
```java
final var codec = Codecs.ofSubSet(items);
final var bestItems = codec.decode(best.genotype());
final var fillSize = bestItems.stream()
	.mapToDouble(it -> it.size)
	.sum();

System.out.println(String.format("Used size of knapsack is %s of available %s.", fillSize, kssize));
System.out.println(String.format("Which is a ratio of %.2f %%. ", 100*fillSize/kssize));
```
Which prints
```
Genotype of best item: [01110110|01011110]
Used size of knapsack is 490.77736663055373 of available 500.0.
Which is a ratio of 98.16 %. 
```

Reply:
Thanks for your reply!
Well, speaking frankly a little bit embarrassing for me ...
I simply say it was due to the late time ...
Anyway, thanks for your update and correction.
Integrated it in the pull request so you can merge it if you want.

Reply:
Thanks, I already integrated this change into the r6.2.0. branch. Because the example was not that clear as it should be.

Reply:
Alright, then I close the pull request.